src/app/frontend/common/components/allocationchart/component.ts

```diff
@@ -78,7 +74,11 @@ export class AllocationChartComponent implements OnInit {
                       .labelType(labelFunc);
 
     chart.tooltip.enabled(this.enableTooltips);
-    chart.tooltip.contentGenerator((obj) => {
+    chart.tooltip.contentGenerator(obj => {
+      if (obj.data.key.includes(':')) {
```

Comment:
What is this logic?

Reply:
> I'd prefer to see the hover behavior seen on [nvd3's site](http://nvd3.org/examples/pie.html)

@jeefy asked for a similar look in the link above. The `key` is of the type `label: count` eg: `Success: 10` and the `value` contains percentage eg: 100, so I could not use value directly. I used the key and split it to achieve the look.
If `:` is present in the key, I'm assuming it is of the form `label: count` and it'll split it, otherwise the old behaviour will be followed.

Reply:
Could you add short comment as this is not obvious what it does?

Reply:
Added.